.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The branch to checkout when cutting the release."
+        required: true
+        default: "main"
+      releaseVersion:
+        description: "Default version to use when preparing a release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Default version to use for new local working copy."
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+
+jobs:
+  build:
+    uses: jqassistant-tooling/jqassistant-github-actions/.github/workflows/release.yml@main
+    with:
+        branch: ${{ github.event.inputs.branch }}
+        releaseVersion: ${{ github.event.inputs.releaseVersion }}
+        developmentVersion: ${{ github.event.inputs.developmentVersion }}
+    secrets:
+      ossrh_username: ${{ secrets.OSSRH_USERNAME }}
```

Comment:
Secrets need to be checked and/or reported from this other PR: https://github.com/jqassistant-plugin/jqassistant-jmolecules-plugin/pull/13#discussion_r1346487169